index.html

```diff
@@ -7,39 +7,74 @@
-        <button>Show answer</button>
-        <p>Answer text</p>
-        <ul>
-          <li>Tag 1</li>
-          <li>Tag 2</li>
-          <li>Tag 3</li>
-          <li>Tag 4</li>
+    <main class="app-grid__scroll-area">
+      <section class="card">
+        <button class="card__bookmark">Bookmark</button>
+        <h2 class="card__heading">Question:</h2>
+        <p class="card__question">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <details>
+          <summary class="card__summary">Show answer</summary>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </details>
+        <ul class="card__ul">
+          <li class="card__ul-li">Tag 1</li>
+          <li class="card__ul-li">Tag 2</li>
+          <li class="card__ul-li">Tag 3</li>
+          <li class="card__ul-li">Tag 4</li>
         </ul>
       </section>
 
-      <section>
-        <button>Bookmark</button>
-        <h2>Question</h2>
-        <p>Question text</p>
-        <button>Show answer</button>
-        <p>Answer text</p>
-        <ul>
-          <li>Tag 1</li>
-          <li>Tag 2</li>
-          <li>Tag 3</li>
-          <li>Tag 4</li>
+      <section class="card">
+        <button class="card__bookmark">Bookmark</button>
+        <h2 class="card__heading">Question:</h2>
+        <p class="card__question">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <details>
+          <summary class="card__summary">Show answer</summary>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </details>
+        <ul class="card__ul">
+          <li class="card__ul-li">Tag 1</li>
+          <li class="card__ul-li">Tag 2</li>
+          <li class="card__ul-li">Tag 3</li>
+          <li class="card__ul-li">Tag 4</li>
+        </ul>
+      </section>
+
+      <section class="card">
+        <button class="card__bookmark">Bookmark</button>
+        <h2 class="card__heading">Question:</h2>
+        <p class="card__question">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Modi earum
+          voluptas quisquam numquam aut nostrum, voluptatum voluptatibus nobis.
+        </p>
+        <details>
+          <summary class="card__summary">Show answer</summary>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </details>
+        <ul class="card__ul">
```

Comment:
Why you wanna hide the answers?
give the ul a role="list" for accessibility.

Reply:
isn´t it part of the app that the answer appear and disappear by clicking the button?